pwiz_tools/Skyline/Model/DocSettings/TransitionSettings.cs

```diff
@@ -1686,7 +1687,18 @@ public bool HasMinIonCount(TransitionGroupDocNode nodeGroup)
             // CONSIDER: Maybe we should be storing a ChromSource on the transition at the time it is created, which
             // seems to be the only time we really know that a precursor ion is being created for MS1 filtering
             // or not.
-            return nodeGroup.Transitions.Count(nodeTran => !nodeTran.IsMs1 || nodeTran.HasLibInfo) >= MinIonCount;
+            if (nodeGroup.Transitions.Count(nodeTran => !nodeTran.IsMs1 || nodeTran.HasLibInfo) >= MinIonCount)
+            {
+                return true;
+            }
+
+            // Special case with feature finding where libraries don't have fragment info, just precursors
+            if (nodeGroup.HasLibInfo && nodeGroup.LibInfo.ScoreType.Equals(ScoreType.HARDKLOR_IDOTP) && nodeGroup.Transitions.All(nodeTran => nodeTran.IsMs1))

```

Comment:
Looks good. Nice effort to minimize the impact. The comment might be improved with a bit more explanation, e.g. "So, having only MS1 transitions is treated as having the minimum ion count."

Reply:
Yeah, could be better:
            // Special case with feature finding where libraries don't have fragment info, just precursors - so a minumum
            // fragment count isn't applicable. Only current scenario for this is the Hardklor-based feature finding.
            // N.B. there is a possibility of the feature finder eventually supporting fragment ions, in which case this needs
            // to be revisited.